client/src/rpc_client.rs

```diff
@@ -764,6 +764,18 @@ impl RpcClient {
         self.send(RpcRequest::GetInflationRate, Value::Null)
     }
 
+    pub fn get_inflation_reward(
+        &self,
+        addresses: &[Pubkey],
+        epoch: Epoch,
```

Comment:
The epoch is optional right?
```suggestion
        epoch: Option<Epoch>,
```